components/collective-navbar/index.js

```diff
@@ -639,4 +688,4 @@ CollectiveNavbar.defaultProps = {
   },
 };
 
-export default React.memo(CollectiveNavbar);
+export default React.memo(withRouter(CollectiveNavbar));
```

Comment:
:ice_cream: For functional components there's also the `useRouter` hook, simpler to add